src/Pim/Bundle/CatalogBundle/Resources/elasticsearch/index_configuration.yml

```diff
@@ -150,6 +150,12 @@ mappings:
                     match_mapping_type: 'boolean'
                     mapping:
                         type: 'boolean'
+            -
+                date:
+                    path_match: 'values.*-date.*'
+                    match_mapping_type: 'string'
+                    mapping:
+                        type: 'keyword'
```

Comment:
I don't understand this mapping, don't we want to take advantage of the Date ES type to use some Date like operators on it ? Can you explain why you map it to a keyword (string) type ?